apps/forums/models.py

```diff
@@ -10,6 +10,9 @@ class Forum(models.Model):
     """Forum represents a message forum
     """
     name = models.CharField(max_length=64)
+    club = models.ForeignKey(
+        Club, related_name='club', on_delete=models.CASCADE
+    )
```

Comment:
## Reviewer Note
Refer to 
https://bawshuman.slack.com/archives/C06DEHYHRB7/p1712615300183329?thread_ts=1712614455.737159&cid=C06DEHYHRB7